uvloop/handles/basetransport.pyx

```diff
@@ -88,10 +90,12 @@ cdef class UVBaseTransport(UVSocketHandle):
         if self._protocol_paused and size <= self._low_water:
             self._protocol_paused = 0
             try:
+                # We need to hold a reference here because context.run() won't
+                resume_writing = self._protocol.resume_writing
                 # We're copying the context to avoid entering context twice,
                 # even though it's not always necessary to copy - it's easier
                 # to copy here than passing down a copied context.
-                self.context.copy().run(self._protocol.resume_writing)
+                self.context.copy().run(resume_writing)
```

Comment:
I propose to do this differently. Create a helper function "run_in_context" and use it like

```
run_in_context(
    self.context.copy(),
    self._protocol.resume_writing
)
```

Reply:
That new function then should consistently be used across the uvloop codebade.